### ISSUE TITLE
Investigate way to make bullet not see N+1 query in post policy

### ISSUE THREAD

**Issue by joshsmith** [code-corps/deprecated-code-corps-rails-api#177]
For some reason, in `PostPolicy`, doing `post.project.organization` registers an N+1 query. We should find out if this is actually an N+1 query, and if not, how to make Bullet not register it as one.

For now, we have been using `around_action` with a `skip_bullet` in the controller. This is not ideal.

**Comment by joshsmith** [comment]
This is now also a problem in GithubRepositories API spec. I'm going to apply the same strategy to `PostPolicy` and not have the `around_action`.

**Comment by joshsmith** [comment]
That strategy specifically is:

```
  before do
    Bullet.enable = false
  end

  after do
    Bullet.enable = true
  end
```

**Comment by begedin** [comment]
I fixed both issues as part of #176.

The commits in question are:
- https://github.com/code-corps/code-corps-api/commit/240d6d6365c95c0b18bdebca6cc0fc79b5ca9aab
- https://github.com/code-corps/code-corps-api/commit/e03383b3f876a9f62dbf6fb5c7f07ed4e8ec1d2d

**Comment by joshsmith** [comment]
:+1:

**Comment by joshsmith** [comment]
Closed by #176
